Formula/minikube.rb

```diff
@@ -0,0 +1,16 @@
+class Minikube < Formula
+  desc "Run Kubernetes locally"
+  homepage "https://github.com/kubernetes/minikube"
+  url "https://github.com/kubernetes/minikube/releases/download/v0.17.1/minikube-darwin-amd64"
```

Comment:
We need to build from source here.

Reply:
Sorry for my ignorance, couldn't we do something like `OS.mac? ? MAC_URL : LINUX_URL`. Or is there a reason to build from source other than OS compatibility?

Reply:
The formula must build from source.